index.rst

```diff
@@ -10,20 +10,867 @@
+
+
+.. [#] A single full focal plane analysis currently takes ~3 min with 2 cores per chip. Note that Rapid Analysis does not need to be run on these images, thus saving compute time, but it is important to make sure the processes are setup such that they do not compete.
+
+.. RA triggers same as PP.
+.. Working on getting data processed via SFP.
+.. AOS trigger could be by via OCPS but could be different.
+
+
+.. _Deliverable 6:
+
+Deliverable 6: Camera Visualization Tool Expansion Support
+-----------------------------------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+  6. Develop a plan and scope estimate to expand the Camera Visualization Tool to support the full commissioning effort.
+
+     This includes identifying libraries/packages/dependencies that require improvements (e.g. Seadragon) and fully scoping what is required to implement the tool with DM tooling such as the Butler.
+     The scope estimate may propose the use of in-kind contribution(s) to this effort if and where applicable.
+
+We have developed a plan to address the visualization requirements documented as part of the FAFFv1 report and further refined based on
+discussion during FAFFv2. This plan is based on use of the Camera Image Visualization (CVT) tool already in use for AuxTel, ComCam and Main Camera.
+
+.. figure:: images/cvt/auxtel.png
+
+   Screenshot of the CVT displaying a recent AuxTel on-sky image.
+
+.. figure:: images/cvt/comcam.png
+
+   Screenshot of the CVT displaying an emulated ComCam image.
+
+.. figure:: images/cvt/MainCamera.png
+
+   Screenshot of the CVT displaying a full focal-plane (dark) image taken during EO testing at SLAC.
+
+Future plans include work under the following major categories:
+
+1. Requirements that can be implemented with existing/planned camera/contributed labor
+2. Requirements which require additional hardware at USDF to support
+3. Requirements which will need significant front-end work
+4. Requirements which require significant DM expertise/assistance
+
+In the last six months, significant progress has been made on category 1, including effort contributed by Oxford,UK under UKD-UKD-S7.
+Progress has also been made on item 4, in particular targeting an early proof-of-concept of displaying DM generated FITS files including with
+instrument signature removal (ISR)
+using the RubinTV generated files from AuxTel. This work is being generalize to ComCam and the main camera,
+with the intention of displaying ISR files generated on the commissioning cluster (Antu -- see above) in addition to raw images from the camera.
+
+This work is being further tracked under: https://jira.lsstcorp.org/browse/SITCOM-190 which includes links to the detailed phased implmentation plans and an
```

Comment:
There is another way to format links and implementation has a typo :)
Consider:
This work is being further tracked under `SITCOM-190 <https://jira.lsstcorp.org/browse/SITCOM-190>`_ which includes links to the detailed phased implementation plans and an evolving set of implementation JIRAs.

Reply:
Done